projects/solar/test/test_relay_fsm.c

```diff
@@ -23,35 +24,62 @@
 
 #define IRRELEVANT_EVENT NUM_SOLAR_RELAY_EVENTS
 
-#define PROCESS_RELAY_FSM_EVENT(storage, e)    \
-  ({                                           \
-    MS_TEST_HELPER_AWAIT_EVENT(e);             \
-    relay_fsm_process_event(&(storage), &(e)); \
-    MS_TEST_HELPER_ASSERT_NO_EVENT_RAISED();   \
+#define PROCESS_RELAY_FSM_OPEN_EVENT(storage, e)                      \
+  ({                                                                  \
+    MS_TEST_HELPER_AWAIT_EVENT(e);                                    \
+    relay_fsm_process_event(&(storage), &(e));                        \
+    MS_TEST_HELPER_CAN_TX_RX(SOLAR_CAN_EVENT_TX, SOLAR_CAN_EVENT_RX); \
+    TEST_ASSERT_TRUE(relay_6_mppts_rx_cb_called);                     \
+    MS_TEST_HELPER_ASSERT_NO_EVENT_RAISED();                          \
+  })
+#define PROCESS_RELAY_FSM_OTHER_EVENT(storage, e) \
+  ({                                              \
+    MS_TEST_HELPER_AWAIT_EVENT(e);                \
+    relay_fsm_process_event(&(storage), &(e));    \
+    MS_TEST_HELPER_ASSERT_NO_EVENT_RAISED();      \
   })
-
 static CanStorage s_can_storage;
 static RelayFsmStorage s_storage;
 
-static bool relay_err_rx_cb_called;
-static StatusCode prv_err_rx_cb(const CanMessage *msg, void *context, CanAckStatus *ack_reply) {
-  relay_err_rx_cb_called = true;
+static bool relay_error_rx_cb_called;
+static bool relay_5_mppts_rx_cb_called;
+static bool relay_6_mppts_rx_cb_called;
+static uint8_t error_array[5];
+static int current_error_num;
```

Comment:
Nit: these should all be prefixed with `s_` (and `current_error_num` shouldn't be `int` but a fixed-width type).

Reply:
done